start_up.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+
+import settings
+import classes
+
+def main():
+    with open(sys.argv[1]) as file:
+        solar_car_processes = [classes.SolarCarProcess(path) for path in file.read().splitlines()]
```

Comment:
I don't think we need the `classes` part here, `SolarCarProcess(path)` should be sufficient.